### ISSUE TITLE
Finetuning Model

### ISSUE THREAD

**Issue by apurvvarshney** [facebookresearch/nwm#5]
Hi, thanks for releasing the code, excited to test it! I had some questions that need clarification:

1.  How was the unlabeled training done on ego4D, I have a similar dataset that I want to test your model on so wondering if you have a finetuning code or the training code that was used for ego4D.
2.  Some steps in the documentation regarding how to evaluate the model on a new dataset would be appreciated with some example config files.

**Comment by amirbar** [comment]
Hello,

Thanks for the interest in our work!
 
1. For training on unlabeled videos, the main idea we used is not to include the navigation action in this summation:https://github.com/facebookresearch/nwm/blob/main/models.py#L239 

2. Will try to add some instructions soon

**Comment by booksource1** [comment]
Hi, thanks for releasing the code.

I’d like to know whether the training on Ego4D was performed after finishing training on the original labeled dataset.
Was the training configuration the same as when training with labels, or did you use any additional settings?

**Comment by amirbar** [comment]
> I’d like to know whether the training on Ego4D was performed after finishing training on the original labeled dataset.

no, there was a single training phase. minibatches contained both labeled and unlabeled data.

> Was the training configuration the same as when training with labels, or did you use any additional settings?

yes, it was almost exactly the same. the only difference was ~2-3 times longer training.
